test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/DefaultProjectSnapshotManagerTest.cs

```diff
@@ -17,6 +17,10 @@ public class DefaultProjectSnapshotManagerTest : ForegroundDispatcherWorkspaceTe
     {
         public DefaultProjectSnapshotManagerTest()
         {
+            // Force VB and C# to Load
+            GC.KeepAlive(typeof(Microsoft.CodeAnalysis.CSharp.SyntaxFactory));
+            GC.KeepAlive(typeof(Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory));
```

Comment:
This is determinism we probably should have had before. This test requires us to have both of these things loaded before creating the workspace.